src/Sylius/Behat/Context/Ui/Admin/DashboardContext.php

```diff
@@ -29,11 +30,14 @@ public function __construct(DashboardPageInterface $dashboardPage)
     }
 
     /**
-     * @When I open administration dashboard
+     * @When I (try to )open administration dashboard
      */
     public function iOpenAdministrationDashboard()
```

Comment:
WDYT about? 
```php
    /**
     * @When I open administration dashboard
     */
    public function iOpenAdministrationDashboard()
    {
        $this->dashboardPage->open();
    }
    /**
     * @When I try to open administration dashboard
     */
    public function iTryToOpenAdministrationDashboard()
    {
        try {
            $this->iOpenAdministrationDashboard();
        } catch (UnexpectedPageException $e) {
        }
    }
```
So we can decide, if we want to catch error or not

Reply:
Yep and moreover i've recently seen there is a tryToOpen method on page object.